#!/usr/bin/env python3
PACKAGE = "reconfigure"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("FR_hip_cur", double_t, 0, "FR hip cur", 0.00, -18, 18)
gen.add("FR_thigh_cur", double_t, 0, "FR thigh cur", 0.00, -18, 18)
gen.add("FR_calf_cur", double_t, 0, "FR calf cur", 0.00, -18, 18)

gen.add("FL_hip_cur", double_t, 0, "FL hip cur", 0.00, -18, 18)
gen.add("FL_thigh_cur", double_t, 0, "FL thigh cur", 0.00, -18, 18)
gen.add("FL_calf_cur", double_t, 0, "FL calf cur", 0.00, -18, 18)

gen.add("BR_hip_cur", double_t, 0, "BR hip cur", 0.00, -18, 18)
gen.add("BR_thigh_cur", double_t, 0, "BR thigh cur", 0.00, -18, 18)
gen.add("BR_calf_cur", double_t, 0, "BR calf cur", 0.00, -18, 18)

gen.add("BL_hip_cur", double_t, 0, "BL hip cur", 0.00, -18, 18)
gen.add("BL_thigh_cur", double_t, 0, "BL thigh cur", 0.00, -18, 18)
gen.add("BL_calf_cur", double_t, 0, "BL calf cur", 0.00, -18, 18)

exit(gen.generate(PACKAGE, "reconfigure", "current_reconfigure"))